proto/pom.xml

```diff
@@ -46,28 +46,45 @@
         </dependency>
     </dependencies>
 
-    <profiles>
-        <profile>
-            <id>build-protobuf</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.xolstice.maven.plugins</groupId>
-                        <artifactId>protobuf-maven-plugin</artifactId>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.os72</groupId>
+                <artifactId>protoc-jar-maven-plugin</artifactId>
+                <version>3.3.0</version>
```

Comment:
Let's move the version out of this pom, to the `plugins-parent`, where all the other plugins are defined. 
BTW, it makes sense to remove old plugin from `plugins-parent` as well as related version property, if it is not used any more (which I believe is the case).